Sources/Plasma/Apps/plClient/plClientLoader.h

```diff
@@ -85,7 +85,7 @@ class plClientLoader : private hsThread
     /**
      * Sets the client display handle.
      */
-    void SetClientDisplay(hsWindowHndl hDC) { fDisplay = hDC; }
+    void SetClientDisplay(hsDisplayHndl hDC) { fDisplay = hDC; }
```

Comment:
You mentioned `HMONITOR` for Windows, but in the way the OpenGL code uses this, it's expected to be the `HDC` device context